cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,15 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  const EMAIL = 'gemowerez@mailinator.com';
+  const USERNAME = 'Ullamco';
+  const PASSWORD = 'qwert12345';
 
+  it('successfully login', () => {
```

Comment:
```suggestion
  it('should have an ability to log in user', () => {
```
Try to name your test in a way to create single sentence when concatenating description + test names, because now you have -  Sign In page successfully login